engine/apps/public_api/views/integrations.py

```diff
@@ -58,17 +59,27 @@ def get_object(self):
             raise NotFound
 
     def perform_update(self, serializer):
-        prev_state = serializer.instance.insight_logs_serialized
+        instance = serializer.instance
+
+        prev_state = instance.insight_logs_serialized
+        prev_web_title_template = instance.web_title_template
+
         serializer.save()
-        new_state = serializer.instance.insight_logs_serialized
+
+        new_state = instance.insight_logs_serialized
+        new_web_title_template = instance.web_title_template
+
         write_resource_insight_log(
-            instance=serializer.instance,
+            instance=instance,
             author=self.request.user,
             event=EntityEvent.UPDATED,
             prev_state=prev_state,
             new_state=new_state,
         )
 
+        if new_web_title_template != prev_web_title_template:
+            update_verbose_name_for_alert_receive_channel.delay(instance.pk)
```

Comment:
Nice idea about delay using PK.

Reply:
I don't like re-rendering all old Alert Groups even in the background. It will cause pretty unpredictable impact on infrastructure. Didn't we agree on leaving them with old cache and speaking about it in the UI?

Reply:
Removed cache invalidation, let's try this without it.